devices/xiaomi.js

```diff
@@ -1381,6 +1381,24 @@ module.exports = [
         },
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['lumi.curtain.acn003'],
+        model: 'ZNCLBL01LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara curtain driver E1',
+        fromZigbee: [fz.xiaomi_curtain_position, fz.battery, fz.cover_position_tilt, fz.xiaomi_curtain_acn003_motor_state],
```

Comment:
Accordingly, now you need to replace `fz.xiaomi_curtain_acn003_motor_state` with `fz.aqara_opple`.